storcli.py

```diff
@@ -89,17 +266,16 @@ def handle_sas_controller(response):
 def handle_megaraid_controller(response):
     (controller_index, baselabel) = get_basic_controller_info(response)
 
-    # BBU Status Optimal value is 0 for cachevault and 32 for BBU
-    add_metric('battery_backup_healthy', baselabel,
-               int(response['Status']['BBU Status'] in [0, 32]))
+    # BBU Status value is not explained in the schema, see note in the header.
+    add_metric('bbu_status', baselabel,
+               int(response['Status']['BBU Status']))
```

Comment:
Cards with no BBU return the string "NA" for BBU Status, so this needs wrapping with try/except.

Reply:
Oh sorry, I missed that one from your commit(s). I'm suspecting this will actually apply to all metrics actually, but we'll see.